web-common/package.json

```diff
@@ -36,9 +36,11 @@
     "@tanstack/svelte-table": "^8.9.5",
     "@tanstack/svelte-virtual": "^3.0.1",
     "@testing-library/svelte": "^4.0.0",
+    "@types/chroma-js": "^2.4.3",
     "@vitest/coverage-c8": "^0.31.0",
     "autoprefixer": "^10.4.13",
     "bits-ui": "^0.11.8",
+    "chroma-js": "^2.4.2",
     "clsx": "^2.0.0",
     "colorjs.io": "^0.4.5",
```

Comment:
can we use either colorjs.io or chroma-js? I don't think we need both.

Reply:
chroma-js has a method that was not in colorjs.io. I can remove colorjs.io